Main.java

```diff
@@ -9,7 +11,39 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // creates a Scanner used for input
+    Scanner input = new Scanner(System.in);
+
+    // Ask user for first point
+    System.out.println("Please enter in the x coordinate of the first point");
+
+    // Declare and initialize a variable for the user to put in a number
+    int x1 = input.nextInt();
+
+    // Ask user for second point
+    System.out.println("Please enter in the y coordinate of the first point");
+
+    // Declare and initialize a variable for the user to put in a number
+    int y1 = input.nextInt();
+
+    // Ask user for third point
+    System.out.println("Please enter in the x coordinate of the second point");
+
+    // Declare and initialize a variable for the user to put in a number
+    int x2 = input.nextInt();
+
+    // Ask user for third point
+    System.out.println("Please enter in the y coordinate of the second point");
+
+    // Declare and initialize a variable for the user to put in a number
+    int y2 = input.nextInt();
+
+    // calculate slope
+    double slope = (y2 - y1) / (x2 - x1);
+
+    // Tell user the slope
```

Comment:
This is great work. My only question would be "can the x or y value contain a decimal place?". They probably should be doubles instead of ints. Otherwise, this is excellent work!